flake8_quotes/__init__.py

```diff
@@ -2,12 +2,17 @@
 import tokenize
 import warnings
 
-import pep8
+import flake8
 
 from flake8_quotes.__about__ import __version__
 
+FLAKE8_VERSION2 = int(flake8.__version__.split('.')[0]) == 2
+if FLAKE8_VERSION2:
+    # Use the pep8/pycodestyle package used by Flake8
+    from flake8.engine import pep8
```

Comment:
Instead of sniffing version from `flake8.__version__`, maybe we can use `try: ... except ImportError:` to determine if the `pep8` engine exists?